### PR TITLE
Remove code for management UI

### PR DESCRIPTION
**Pull request by CarstenWalther**
Remove code that was moved to the management UI service in this PR: https://github.com/hpi-epic/pricewars-mgmt-ui/pull/13

Restrict merchant access to kafka topic. Now they only can request market situations and sales.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # Kafka Reverse Proxy
 
-This repository contains the Kafka Reverse Proxy which acts as an interfacing components between our Kafka streams and the components that need access to it. These components are:
+This repository contains the Kafka Reverse Proxy which acts as an interfacing component between our Kafka streams and merchants.
+Data-driven merchants have to be able to retrieve past market situations written to Kafka to do their learning. However, they should only be allowed to access the market situation data that they would have access to on a real system as well, ie the offers that were on the marketplace at a certain time and only their own sales data.
```
Comment (Bouncner) [inline README.md]: `, ie` >> `, i.e., `

**Comment by Bouncner** [comment]
I'll run some tests on the benchmark server.

File: LoggerApp.py [hunk 2]
```diff
@@ -198,29 +124,14 @@ def calculate_id(token: str) -> str:
     return base64.b64encode(hashlib.sha256(token.encode('utf-8')).digest()).decode('utf-8')
 
 
-def wait_for_kafka(kafka_endpoint, timeout: float = 60) -> None:
```
Comment (Bouncner) [inline LoggerApp.py]: Why can we remove this waiting for kafka?

Comment (CarstenWalther) [reply LoggerApp.py]: This service can handle if kafka is down. In that case it will send an error to the requester.

**Pull request merged**